CMakeLists.txt

```diff
@@ -122,3 +122,20 @@ add_subdirectory(extras)
 # Install the export file specifying all the targets for RapidCheck
 install(EXPORT rapidcheckConfig DESTINATION share/rapidcheck/cmake)
 export(EXPORT rapidcheckConfig FILE rapidcheckConfig.cmake)
+
+set(PKG_CONFIG_REQUIRES)
+set(PKG_CONFIG_DESCRIPTION_SUMMARY "C++ framework for property based testing inspired by QuickCheck and other similar frameworks")
+set(PKG_CONFIG_VERSION)
+set(PKG_CONFIG_LIBDIR "\${prefix}/lib")
+set(PKG_CONFIG_INCLUDEDIR "\${prefix}/include")
+set(PKG_CONFIG_LIBS)
```

Comment:
Is `PKG_CONFIG_LIBS` intentionally empty?

Reply:
Yes I think so, otherwise it won't be replaced.

Reply:
I'm not sure this is correct. rapidcheck ships a `librapidcheck.so` file which doesn't get picked up with `--libs` because this is empty

Reply:
I think the correct value for `PKG_CONFIG_LIBS` should be ` "-L\${libdir} -lrapidcheck"`

Reply:
Oh nevermind that's not correct. @octurion is right. Sorry!

Reply:
https://github.com/emil-e/rapidcheck/pull/326 now intends to apply @octurion's suggestion.